examples/oidc/james/imapserver.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<imapservers>
+    <imapserver enabled="true">
+        <jmxName>imapserver-ssl</jmxName>
+        <bind>0.0.0.0:993</bind>
```

Comment:
Hmmm... the port 993 is usually for SSL. Clearly we not gonna bother do SSL for an example running locally in a docker-compose but if using TLS shouldn't we use the port 143 for this instead then?

Reply:
https://github.com/apache/james-project/pull/847#discussion_r786448837

Maybe we use this ssl port as an example for one wants more security when using OIDC?

Reply:
Your smtp port is the tls one btw (587... ssl is 465^^)

I'm just a bit confuse with the new conf actually... 

Now we have the `auth` block with the param `requireSSL`, while I think before we were checking on the socketTLS in tls block for ssl. Does that param in the tls block still make sense or not? What is the difference? cc @chibenwa

Reply:
> Does that param in the tls block still make sense or not?

On the SSL SMTP server this makes little sense to check ssl is activated as it will always be.

We can safely remove `requireSSL`